data-pipeline-examples/src/main/java/org/deeplearning4j/datapipelineexamples/utils/DownloaderUtility.java

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2019 Skymind, Inc.
+ *
```

Comment:
Please check the License header on all new and changed files whether it needs updating.

Reply:
Added copyrights